### PR TITLE
fix: flow navigation alignments

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

This PR brings back some alignments from old version until some more in depth design work will be made on FN

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/components/src/flowNavigation/FlowNavigation.js [hunk 1]
```diff
@@ -53,11 +59,16 @@ const FlowNavigation = ({ activeStep, avatar, logo, onClose, onGoBack, done, ste
...
-          'np-flow-navigation--large': !isSmall,
-          'np-flow-navigation--small': isSmall,
-        })}
+        className={classNames(
+          'np-flow-navigation__content p-x-3',
+          {
+            'np-flow-navigation--hidden': !clientWidth,
+            'xs-max': clientWidth < Breakpoint.SMALL,
+          },
+          Object.keys(containerBreakpoints).filter(
```
Comment (fergusjordantw) [inline packages/components/src/flowNavigation/FlowNavigation.js:68]: Just a thought but should these be prefixed by `np-flow-navigation--` for namespacing safety?

File: packages/components/src/flowNavigation/FlowNavigation.js [hunk 2]
```diff
@@ -12,12 +12,18 @@ import { useClientWidth } from '../common/hooks';
 
 import './FlowNavigation.css';
 
+// Size switches on parent container which may or may not have the same size as the window.
+const containerBreakpoints = {
+  sm: Breakpoint.SMALL,
+  lg: Breakpoint.LARGE,
+  xl: Breakpoint.EXTRA_LARGE,
```
Comment (fergusjordantw) [inline packages/components/src/flowNavigation/FlowNavigation.js:19]: With this being 1200 and a max-width of 1164, won't this never be the case?

**Review by fergusjordantw — COMMENTED** [review]
Small things

File: packages/components/src/flowNavigation/FlowNavigation.js (hunk 2 above)
Comment (andreapiras) [reply packages/components/src/flowNavigation/FlowNavigation.js:19]: This is based on the parent container that currently is 100% of window

File: packages/components/src/flowNavigation/FlowNavigation.js (hunk 1 above)
Comment (andreapiras) [reply packages/components/src/flowNavigation/FlowNavigation.js:68]: Yup makes sense, they are still defined inside np-flow-navigation. I wanted to eventually make them reusable but it probably a bit premature

**Review by fergusjordantw — APPROVED** [review]
Looks good

**Pull request merged**